lib/assessment.rb

```diff
@@ -5,37 +5,42 @@ class Assessment
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
   def longest(enum)
-    # Your code goes here
+    enum.max { |a, b| a.length <=> b.length }
   end
 
   # Question 2
   # Write ruby code to return the sum of entries in the
-  # enumerable supplied (i.e. apply the `+` operator)
+  # enumerable supplied
+  #(i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+# What enumerable supplied? Is there something we should actually be working off of?
+  enum.sum #or use the inject method?
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    !numbers.any? &block
   end
+  print numbers
 
   # Question 4
   # Write ruby code that returns true if every entry in the
   # numbers enumerable are even
   def every_even(numbers)
-    # Your code goes here
+    numbers.all? &block
   end
+  print numbers
 
   # Question 5
   # Write ruby code that returns an array with all of the
   # entries from the words enumerable capitalized.
   def transform(words)
-    # Your code goes here
+    words.capitalized
```

Comment:
You need to iterate through all of the words and apply a transformation to them (specifically, capitalization). What enumerable methods do you think you might use to accomplish that?